Regression/Checksum/benchmarks_json/Python_LaserAccelerationMR.json

```diff
@@ -1,25 +1,46 @@
 {
+  "beam": {
```

Comment:
Keep in mind that this is actually a new, different test, but shows up as a benchmark change because an old test with the same name (but actually different setup) was removed (see other comment on WarpX-tests.ini).

Reply:
Do you maybe want to rename that benchmark file & section to avoid the confusion and make it consistent with 1D/3D/RZ naming?